ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-bgp.md

```diff
@@ -1623,12 +1644,20 @@
                 route_targets:
                   - <str>
                 route_map: <str>
+
+                # RCF function name with parenthesis.
+                # Example: MyFunction(myarg).
+                rcf: <str>
```

Comment:
There is additional RCF support for vpn-ipv4/ipv6 address families 
```
wan1-dc1(config-router-bgp-vrf-Mahesh)#route-target import vpn-ipv4 rcf FUN1() ?
  vpn-route  RCF filter VPN routes
  <cr>       

wan1-dc1(config-router-bgp-vrf-Mahesh)#route-target import vpn-ipv4 rcf FUN1() vpn-route ?
  filter-rcf  RCF filter routes

wan1-dc1(config-router-bgp-vrf-Mahesh)#route-target import vpn-ipv4 rcf FUN1() vpn-route filter-rcf FUN2() ?
  <cr>  
```
Also RCF not supported for flow-spec-vpn-pv4/ipv6 address families

Reply:
Added!